proto/v1/message.proto

```diff
@@ -16,7 +16,7 @@ message Auction {
 
 message Bid {
     string auction_id = 1;
-    string miner_addr = 2;
+    string storage_provider_id = 2;
```

Comment:
I hesitated to do this last time but renaming a field is in fact not a back incompatible change.